eng/common/pipelines/templates/steps/docsms-ensure-validation.yml

```diff
@@ -9,3 +9,4 @@ steps:
       exit 1
     }
   displayName: Check package validation results
+  condition: and(succeeded(), ne(variables['DocsMsPackagesAllValid'], ''))
```

Comment:
Are we sure it is always set to something non-empty in other cases?

Reply:
Yes. When `Update-DocsMsMetadata.ps1` runs it sets the variable to either `true` or `false`: https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/Update-DocsMsMetadata.ps1#L265-L269

Reply:
I thought you just going to set the variable to true in Update-DocsMsMetadata.ps1 if the package list was empty.

Reply:
Closing the loop on our offline discussion: `Update-DocsMsMetadata.ps1` is skipped in cases where there are no packages. https://github.com/Azure/azure-sdk-for-java/blob/djurek/ignore-undocumentable-packages/eng/common/pipelines/templates/steps/update-docsms-metadata.yml#L36-L39